### PR TITLE
NavBar

### PR DESCRIPTION
**Pull request by holgermanzke**
Add NavButtons to NavBar.

### CODE REVIEW

**Comment by tobihoff** [comment]
lgtm

**Comment by holgermanzke** [comment]
Bildschirmfoto 2019-12-20 um 10 07 42

File: client/src/resources/logout.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="36" height="31.5" viewBox="0 0 36 31.5"><defs></defs><path fill="#f4f4f4" d="M13.5,0V4.5h18V27h-18v4.5H36V0ZM9,9,0,15.75,9,22.5V18H27V13.5H9Z"/></svg>
```
Comment (manuelwecker) [inline client/src/resources/logout.svg]: Well cleaned! Do you need `<defs></defs>`

File: client/src/stories/0-Welcome.stories.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
-import React from 'react';
```
Comment (manuelwecker) [inline client/src/stories/0-Welcome.stories.js]: Do you have prettier code rules. Make sure to run them trough all files. See more on slack

File: client/src/resources/logout.svg (hunk 1 above)
Comment (tobiaserich) [inline client/src/resources/logout.svg]: ```<defs></defs>``` is empty, is it really necessary?

**Pull request merged**